[libsodium] Build error on arm64-android

Package: libsodium:arm64-android@1.0.19#1

**Host Environment**

- Host: x64-windows
- Compiler: Clang 17.0.2
-    vcpkg-tool version: 2024-02-07-8a83681f921b10d86ae626fd833c253f4f8c355b
    vcpkg-scripts version: unknown

**To Reproduce**

`vcpkg install `
**Failure logs**

```
-- Using cached jedisct1-libsodium-1.0.19.tar.gz.
-- Cleaning sources at D:/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/src/1.0.19-fdaa0be457.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/sandbox/sandbox/toolchain/vcpkg/downloads/jedisct1-libsodium-1.0.19.tar.gz
-- Using source at D:/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/src/1.0.19-fdaa0be457.clean
-- Getting CMake variables for arm64-android
-- Using cached msys2-autoconf-wrapper-20221207-1-any.pkg.tar.zst.
-- Using cached msys2-automake-wrapper-20221207-2-any.pkg.tar.zst.
-- Using cached msys2-binutils-2.42-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libtool-2.4.7-3-x86_64.pkg.tar.zst.
-- Using cached msys2-make-4.4.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-pkgconf-2.1.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-which-2.21-4-x86_64.pkg.tar.zst.
-- Using cached msys2-bash-5.2.026-1-x86_64.pkg.tar.zst.
-- Using cached msys2-coreutils-8.32-5-x86_64.pkg.tar.zst.
-- Using cached msys2-file-5.45-1-x86_64.pkg.tar.zst.
-- Using cached msys2-gawk-5.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-grep-1~3.0-6-x86_64.pkg.tar.zst.
-- Using cached msys2-gzip-1.13-1-x86_64.pkg.tar.zst.
-- Using cached msys2-diffutils-3.10-1-x86_64.pkg.tar.zst.
-- Using cached msys2-sed-4.9-1-x86_64.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.4.10-4-x86_64.pkg.tar.zst.
-- Using cached msys2-autoconf2.71-2.71-3-any.pkg.tar.zst.
-- Using cached msys2-automake1.16-1.16.5-1-any.pkg.tar.zst.
-- Using cached msys2-libiconv-1.17-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libintl-0.22.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-zlib-1.3.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-tar-1.35-2-x86_64.pkg.tar.zst.
-- Using cached msys2-findutils-4.9.0-3-x86_64.pkg.tar.zst.
-- Using cached msys2-gmp-6.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-gcc-libs-13.2.0-2-x86_64.pkg.tar.zst.
-- Using cached msys2-libbz2-1.0.8-4-x86_64.pkg.tar.zst.
-- Using cached msys2-liblzma-5.4.6-2-x86_64.pkg.tar.zst.
-- Using cached msys2-libzstd-1.5.5-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libreadline-8.2.010-1-x86_64.pkg.tar.zst.
-- Using cached msys2-mpfr-4.2.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libpcre-8.45-4-x86_64.pkg.tar.zst.
-- Using cached msys2-m4-1.4.19-2-x86_64.pkg.tar.zst.
-- Using cached msys2-perl-5.38.2-1-x86_64.pkg.tar.zst.
-- Using cached msys2-ncurses-6.4-2-x86_64.pkg.tar.zst.
-- Using cached msys2-libxcrypt-4.4.36-1-x86_64.pkg.tar.zst.
-- Using msys root at D:/sandbox/sandbox/toolchain/vcpkg/downloads/tools/msys2/09133458fed80fcb
-- Generating configure for arm64-android
-- Finished generating configure for arm64-android
-- Using cached msys2-mingw-w64-x86_64-pkgconf-1~2.1.0-1-any.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.4.10-4-x86_64.pkg.tar.zst.
-- Using msys root at D:/sandbox/sandbox/toolchain/vcpkg/downloads/tools/msys2/fdbea3694fb5c0d4
-- Configuring arm64-android-dbg
-- Configuring arm64-android-rel
-- Building arm64-android-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: D:/sandbox/sandbox/toolchain/vcpkg/downloads/tools/msys2/09133458fed80fcb/usr/bin/make.exe -j 33 --trace -f Makefile all
    Working Directory: D:/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/
    See logs for more information:
      D:\sandbox\sandbox\toolchain\vcpkg\buildtrees\libsodium\build-arm64-android-dbg-out.log
      D:\sandbox\sandbox\toolchain\vcpkg\buildtrees\libsodium\build-arm64-android-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:136 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/libsodium/portfile.cmake:49 (vcpkg_install_make)
  scripts/ports.cmake:172 (include)



```
<details><summary>D:\sandbox\sandbox\toolchain\vcpkg\buildtrees\libsodium\build-arm64-android-dbg-err.log</summary>

```
In file included from ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_aead/aes256gcm/armcrypto/aead_aes256gcm_armcrypto.c:35:
C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/lib/clang/17/include/arm_neon.h:62829:32: error: always_inline function 'vget_high_p64' requires target feature 'crypto', but would be inlined into function 'vmull_high_p64' that is compiled without support for 'crypto'
  __ret = vmull_p64((poly64_t)(vget_high_p64(__p0)), (poly64_t)(vget_high_p64(__p1)));
                               ^
C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/lib/clang/17/include/arm_neon.h:62829:65: error: always_inline function 'vget_high_p64' requires target feature 'crypto', but would be inlined into function 'vmull_high_p64' that is compiled without support for 'crypto'
  __ret = vmull_p64((poly64_t)(vget_high_p64(__p0)), (poly64_t)(vget_high_p64(__p1)));
                                                                ^
2 errors generated.
libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
libtool: warning: '-version-info/-version-number' is ignored for convenience libraries
make[3]: *** [Makefile:2489: crypto_aead/aes256gcm/armcrypto/libarmcrypto_la-aead_aes256gcm_armcrypto.lo] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [Makefile:3397: all-recursive] Error 1
make[1]: *** [Makefile:409: all-recursive] Error 1
make: *** [Makefile:520: all-recursive] Error 1
```
</details>

<details><summary>D:\sandbox\sandbox\toolchain\vcpkg\buildtrees\libsodium\build-arm64-android-dbg-out.log</summary>

```
Makefile:520: update target 'all-recursive' due to: target is .PHONY
fail=; \
if (target_option=k; case ${target_option-} in ?) ;; *) echo "am__make_running_with_option: internal error: invalid" "target option '${target_option-}' specified" >&2; exit 1;; esac; has_opt=no; sane_makeflags=$MAKEFLAGS; if { if test -z '0'; then false; elif test -n 'x86_64-pc-msys'; then true; elif test -n '4.4.1' && test -n '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg'; then true; else false; fi; }; then sane_makeflags=$MFLAGS; else case $MAKEFLAGS in *\\[\ \	]*) bs=\\; sane_makeflags=`printf '%s\n' "$MAKEFLAGS" | sed "s/$bs$bs[$bs $bs	]*//g"`;; esac; fi; skip_next=no; strip_trailopt () { flg=`printf '%s\n' "$flg" | sed "s/$1.*$//"`; }; for flg in $sane_makeflags; do test $skip_next = yes && { skip_next=no; continue; }; case $flg in *=*|--*) continue;; -*I) strip_trailopt 'I'; skip_next=yes;; -*I?*) strip_trailopt 'I';; -*O) strip_trailopt 'O'; skip_next=yes;; -*O?*) strip_trailopt 'O';; -*l) strip_trailopt 'l'; skip_next=yes;; -*l?*) strip_trailopt 'l';; -[dEDm]) skip_next=yes;; -[JT]) skip_next=yes;; esac; case $flg in *$target_option*) has_opt=yes; break;; esac; done; test $has_opt = yes); then \
  failcom='fail=yes'; \
else \
  failcom='exit 1'; \
fi; \
dot_seen=no; \
target=`echo all-recursive | sed s/-recursive//`; \
case "all-recursive" in \
  distclean-* | maintainer-clean-*) list='builds contrib dist-build src test' ;; \
  *) list='builds contrib dist-build src test' ;; \
esac; \
for subdir in $list; do \
  echo "Making $target in $subdir"; \
  if test "$subdir" = "."; then \
    dot_seen=yes; \
    local_target="$target-am"; \
  else \
    local_target="$target"; \
  fi; \
  (CDPATH="${ZSH_VERSION+.}:" && cd $subdir && /usr/bin/make  $local_target) \
  || eval $failcom; \
done; \
if test "$dot_seen" = "no"; then \
  /usr/bin/make  "$target-am" || exit 1; \
fi; test -z "$fail"
Making all in builds
make[1]: Entering directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/builds'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/builds'
Making all in contrib
make[1]: Entering directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/contrib'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/contrib'
Making all in dist-build
make[1]: Entering directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/dist-build'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/dist-build'
Making all in src
make[1]: Entering directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src'
Makefile:409: update target 'all-recursive' due to: target is .PHONY
fail=; \
if (target_option=k; case ${target_option-} in ?) ;; *) echo "am__make_running_with_option: internal error: invalid" "target option '${target_option-}' specified" >&2; exit 1;; esac; has_opt=no; sane_makeflags=$MAKEFLAGS; if { if test -z '1'; then false; elif test -n 'x86_64-pc-msys'; then true; elif test -n '4.4.1' && test -n '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src'; then true; else false; fi; }; then sane_makeflags=$MFLAGS; else case $MAKEFLAGS in *\\[\ \	]*) bs=\\; sane_makeflags=`printf '%s\n' "$MAKEFLAGS" | sed "s/$bs$bs[$bs $bs	]*//g"`;; esac; fi; skip_next=no; strip_trailopt () { flg=`printf '%s\n' "$flg" | sed "s/$1.*$//"`; }; for flg in $sane_makeflags; do test $skip_next = yes && { skip_next=no; continue; }; case $flg in *=*|--*) continue;; -*I) strip_trailopt 'I'; skip_next=yes;; -*I?*) strip_trailopt 'I';; -*O) strip_trailopt 'O'; skip_next=yes;; -*O?*) strip_trailopt 'O';; -*l) strip_trailopt 'l'; skip_next=yes;; -*l?*) strip_trailopt 'l';; -[dEDm]) skip_next=yes;; -[JT]) skip_next=yes;; esac; case $flg in *$target_option*) has_opt=yes; break;; esac; done; test $has_opt = yes); then \
  failcom='fail=yes'; \
else \
  failcom='exit 1'; \
fi; \
dot_seen=no; \
target=`echo all-recursive | sed s/-recursive//`; \
case "all-recursive" in \
  distclean-* | maintainer-clean-*) list='libsodium' ;; \
  *) list='libsodium' ;; \
esac; \
for subdir in $list; do \
  echo "Making $target in $subdir"; \
  if test "$subdir" = "."; then \
    dot_seen=yes; \
    local_target="$target-am"; \
  else \
    local_target="$target"; \
  fi; \
  (CDPATH="${ZSH_VERSION+.}:" && cd $subdir && /usr/bin/make  $local_target) \
  || eval $failcom; \
done; \
if test "$dot_seen" = "no"; then \
  /usr/bin/make  "$target-am" || exit 1; \
fi; test -z "$fail"
Making all in libsodium
make[2]: Entering directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src/libsodium'
Makefile:3397: update target 'all-recursive' due to: target is .PHONY
fail=; \
if (target_option=k; case ${target_option-} in ?) ;; *) echo "am__make_running_with_option: internal error: invalid" "target option '${target_option-}' specified" >&2; exit 1;; esac; has_opt=no; sane_makeflags=$MAKEFLAGS; if { if test -z '2'; then false; elif test -n 'x86_64-pc-msys'; then true; elif test -n '4.4.1' && test -n '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src/libsodium'; then true; else false; fi; }; then sane_makeflags=$MFLAGS; else case $MAKEFLAGS in *\\[\ \	]*) bs=\\; sane_makeflags=`printf '%s\n' "$MAKEFLAGS" | sed "s/$bs$bs[$bs $bs	]*//g"`;; esac; fi; skip_next=no; strip_trailopt () { flg=`printf '%s\n' "$flg" | sed "s/$1.*$//"`; }; for flg in $sane_makeflags; do test $skip_next = yes && { skip_next=no; continue; }; case $flg in *=*|--*) continue;; -*I) strip_trailopt 'I'; skip_next=yes;; -*I?*) strip_trailopt 'I';; -*O) strip_trailopt 'O'; skip_next=yes;; -*O?*) strip_trailopt 'O';; -*l) strip_trailopt 'l'; skip_next=yes;; -*l?*) strip_trailopt 'l';; -[dEDm]) skip_next=yes;; -[JT]) skip_next=yes;; esac; case $flg in *$target_option*) has_opt=yes; break;; esac; done; test $has_opt = yes); then \
  failcom='fail=yes'; \
else \
  failcom='exit 1'; \
fi; \
dot_seen=no; \
target=`echo all-recursive | sed s/-recursive//`; \
case "all-recursive" in \
  distclean-* | maintainer-clean-*) list='include' ;; \
  *) list='include' ;; \
esac; \
for subdir in $list; do \
  echo "Making $target in $subdir"; \
  if test "$subdir" = "."; then \
    dot_seen=yes; \
    local_target="$target-am"; \
  else \
    local_target="$target"; \
  fi; \
  (CDPATH="${ZSH_VERSION+.}:" && cd $subdir && /usr/bin/make  $local_target) \
  || eval $failcom; \
done; \
if test "$dot_seen" = "no"; then \
  /usr/bin/make  "$target-am" || exit 1; \
fi; test -z "$fail"
Making all in include
make[3]: Entering directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src/libsodium/include'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src/libsodium/include'
make[3]: Entering directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src/libsodium'
Makefile:1264: update target 'crypto_aead/aegis128l/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/aegis128l
Makefile:1267: update target 'crypto_aead/aegis128l/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/aegis128l/.deps
: > crypto_aead/aegis128l/.dirstamp
: > crypto_aead/aegis128l/.deps/.dirstamp
Makefile:1273: update target 'crypto_aead/aegis256/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/aegis256
Makefile:1276: update target 'crypto_aead/aegis256/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/aegis256/.deps
: > crypto_aead/aegis256/.deps/.dirstamp
: > crypto_aead/aegis256/.dirstamp
Makefile:1381: update target 'crypto_aead/aes256gcm/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/aes256gcm
Makefile:1384: update target 'crypto_aead/aes256gcm/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/aes256gcm/.deps
: > crypto_aead/aes256gcm/.dirstamp
: > crypto_aead/aes256gcm/.deps/.dirstamp
Makefile:1390: update target 'crypto_aead/chacha20poly1305/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/chacha20poly1305
Makefile:1393: update target 'crypto_aead/chacha20poly1305/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/chacha20poly1305/.deps
: > crypto_aead/chacha20poly1305/.dirstamp
: > crypto_aead/chacha20poly1305/.deps/.dirstamp
Makefile:1399: update target 'crypto_aead/xchacha20poly1305/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/xchacha20poly1305
Makefile:1402: update target 'crypto_aead/xchacha20poly1305/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_aead/xchacha20poly1305/.deps
: > crypto_aead/xchacha20poly1305/.dirstamp
: > crypto_aead/xchacha20poly1305/.deps/.dirstamp
Makefile:1408: update target 'crypto_auth/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_auth
Makefile:1411: update target 'crypto_auth/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_auth/.deps
: > crypto_auth/.dirstamp
: > crypto_auth/.deps/.dirstamp
Makefile:1416: update target 'crypto_auth/hmacsha256/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_auth/hmacsha256
Makefile:1419: update target 'crypto_auth/hmacsha256/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_auth/hmacsha256/.deps
: > crypto_auth/hmacsha256/.dirstamp
: > crypto_auth/hmacsha256/.deps/.dirstamp
Makefile:1425: update target 'crypto_auth/hmacsha512/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_auth/hmacsha512
Makefile:1428: update target 'crypto_auth/hmacsha512/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_auth/hmacsha512/.deps
: > crypto_auth/hmacsha512/.dirstamp
: > crypto_auth/hmacsha512/.deps/.dirstamp
Makefile:1434: update target 'crypto_auth/hmacsha512256/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_auth/hmacsha512256
Makefile:1437: update target 'crypto_auth/hmacsha512256/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_auth/hmacsha512256/.deps
: > crypto_auth/hmacsha512256/.dirstamp
: > crypto_auth/hmacsha512256/.deps/.dirstamp
Makefile:1443: update target 'crypto_box/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_box
Makefile:1446: update target 'crypto_box/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_box/.deps
: > crypto_box/.deps/.dirstamp
: > crypto_box/.dirstamp
Makefile:1457: update target 'crypto_box/curve25519xsalsa20poly1305/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_box/curve25519xsalsa20poly1305
Makefile:1460: update target 'crypto_box/curve25519xsalsa20poly1305/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_box/curve25519xsalsa20poly1305/.deps
: > crypto_box/curve25519xsalsa20poly1305/.dirstamp
: > crypto_box/curve25519xsalsa20poly1305/.deps/.dirstamp
Makefile:1466: update target 'crypto_core/ed25519/ref10/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/ed25519/ref10
Makefile:1469: update target 'crypto_core/ed25519/ref10/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/ed25519/ref10/.deps
: > crypto_core/ed25519/ref10/.dirstamp
: > crypto_core/ed25519/ref10/.deps/.dirstamp
Makefile:1475: update target 'crypto_core/hchacha20/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/hchacha20
Makefile:1478: update target 'crypto_core/hchacha20/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/hchacha20/.deps
: > crypto_core/hchacha20/.dirstamp
: > crypto_core/hchacha20/.deps/.dirstamp
Makefile:1484: update target 'crypto_core/hsalsa20/ref2/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/hsalsa20/ref2
Makefile:1487: update target 'crypto_core/hsalsa20/ref2/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/hsalsa20/ref2/.deps
: > crypto_core/hsalsa20/ref2/.dirstamp
: > crypto_core/hsalsa20/ref2/.deps/.dirstamp
Makefile:1493: update target 'crypto_core/hsalsa20/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/hsalsa20
Makefile:1496: update target 'crypto_core/hsalsa20/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/hsalsa20/.deps
: > crypto_core/hsalsa20/.dirstamp
: > crypto_core/hsalsa20/.deps/.dirstamp
Makefile:1502: update target 'crypto_core/salsa/ref/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/salsa/ref
Makefile:1505: update target 'crypto_core/salsa/ref/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/salsa/ref/.deps
: > crypto_core/salsa/ref/.dirstamp
: > crypto_core/salsa/ref/.deps/.dirstamp
Makefile:1511: update target 'crypto_core/softaes/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/softaes
Makefile:1514: update target 'crypto_core/softaes/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_core/softaes/.deps
: > crypto_core/softaes/.dirstamp
: > crypto_core/softaes/.deps/.dirstamp
Makefile:1520: update target 'crypto_generichash/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_generichash
Makefile:1523: update target 'crypto_generichash/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_generichash/.deps
: > crypto_generichash/.dirstamp
: > crypto_generichash/.deps/.dirstamp
Makefile:1529: update target 'crypto_generichash/blake2b/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_generichash/blake2b
Makefile:1532: update target 'crypto_generichash/blake2b/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_generichash/blake2b/.deps
: > crypto_generichash/blake2b/.dirstamp
: > crypto_generichash/blake2b/.deps/.dirstamp
Makefile:1312: update target 'crypto_generichash/blake2b/ref/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_generichash/blake2b/ref
Makefile:1315: update target 'crypto_generichash/blake2b/ref/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_generichash/blake2b/ref/.deps
: > crypto_generichash/blake2b/ref/.deps/.dirstamp
: > crypto_generichash/blake2b/ref/.dirstamp
Makefile:1547: update target 'crypto_hash/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash
Makefile:1550: update target 'crypto_hash/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash/.deps
: > crypto_hash/.dirstamp
: > crypto_hash/.deps/.dirstamp
Makefile:1555: update target 'crypto_hash/sha256/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash/sha256
Makefile:1558: update target 'crypto_hash/sha256/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash/sha256/.deps
: > crypto_hash/sha256/.dirstamp
: > crypto_hash/sha256/.deps/.dirstamp
Makefile:1564: update target 'crypto_hash/sha256/cp/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash/sha256/cp
Makefile:1567: update target 'crypto_hash/sha256/cp/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash/sha256/cp/.deps
: > crypto_hash/sha256/cp/.dirstamp
: > crypto_hash/sha256/cp/.deps/.dirstamp
Makefile:1573: update target 'crypto_hash/sha512/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash/sha512
Makefile:1576: update target 'crypto_hash/sha512/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash/sha512/.deps
: > crypto_hash/sha512/.dirstamp
: > crypto_hash/sha512/.deps/.dirstamp
Makefile:1582: update target 'crypto_hash/sha512/cp/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash/sha512/cp
Makefile:1585: update target 'crypto_hash/sha512/cp/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_hash/sha512/cp/.deps
: > crypto_hash/sha512/cp/.dirstamp
: > crypto_hash/sha512/cp/.deps/.dirstamp
Makefile:1591: update target 'crypto_kdf/blake2b/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_kdf/blake2b
Makefile:1594: update target 'crypto_kdf/blake2b/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_kdf/blake2b/.deps
: > crypto_kdf/blake2b/.dirstamp
: > crypto_kdf/blake2b/.deps/.dirstamp
Makefile:1600: update target 'crypto_kdf/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_kdf
Makefile:1603: update target 'crypto_kdf/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_kdf/.deps
: > crypto_kdf/.dirstamp
: > crypto_kdf/.deps/.dirstamp
Makefile:1608: update target 'crypto_kdf/hkdf/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_kdf/hkdf
Makefile:1611: update target 'crypto_kdf/hkdf/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_kdf/hkdf/.deps
: > crypto_kdf/hkdf/.deps/.dirstamp
: > crypto_kdf/hkdf/.dirstamp
Makefile:1620: update target 'crypto_kx/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_kx
Makefile:1623: update target 'crypto_kx/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_kx/.deps
: > crypto_kx/.dirstamp
: > crypto_kx/.deps/.dirstamp
Makefile:1628: update target 'crypto_onetimeauth/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_onetimeauth
Makefile:1631: update target 'crypto_onetimeauth/.deps/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_onetimeauth/.deps
: > crypto_onetimeauth/.dirstamp
: > crypto_onetimeauth/.deps/.dirstamp
Makefile:1637: update target 'crypto_onetimeauth/poly1305/.dirstamp' due to: target does not exist
/usr/bin/mkdir -p crypto_onetimeauth/poly1305
Makefile:1640: update target 'crypto_onetimeauth/poly1305/.deps/.dirstamp' due to: target does not exist
...
Skipped 707 lines
...
mv -f crypto_stream/salsa208/.deps/libsodium_la-stream_salsa208.Tpo crypto_stream/salsa208/.deps/libsodium_la-stream_salsa208.Plo
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT crypto_aead/aes256gcm/armcrypto/libarmcrypto_la-aead_aes256gcm_armcrypto.lo -MD -MP -MF crypto_aead/aes256gcm/armcrypto/.deps/libarmcrypto_la-aead_aes256gcm_armcrypto.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_aead/aes256gcm/armcrypto/aead_aes256gcm_armcrypto.c -fPIE -o crypto_aead/aes256gcm/armcrypto/libarmcrypto_la-aead_aes256gcm_armcrypto.o
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT crypto_stream/salsa208/ref/libsodium_la-stream_salsa208_ref.lo -MD -MP -MF crypto_stream/salsa208/ref/.deps/libsodium_la-stream_salsa208_ref.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_stream/salsa208/ref/stream_salsa208_ref.c -fPIE -o crypto_stream/salsa208/ref/libsodium_la-stream_salsa208_ref.o
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT crypto_scalarmult/ristretto255/ref10/libsodium_la-scalarmult_ristretto255_ref10.lo -MD -MP -MF crypto_scalarmult/ristretto255/ref10/.deps/libsodium_la-scalarmult_ristretto255_ref10.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_scalarmult/ristretto255/ref10/scalarmult_ristretto255_ref10.c -fPIE -o crypto_scalarmult/ristretto255/ref10/libsodium_la-scalarmult_ristretto255_ref10.o
mv -f crypto_pwhash/scryptsalsa208sha256/.deps/libsodium_la-pbkdf2-sha256.Tpo crypto_pwhash/scryptsalsa208sha256/.deps/libsodium_la-pbkdf2-sha256.Plo
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT crypto_stream/xchacha20/libsodium_la-stream_xchacha20.lo -MD -MP -MF crypto_stream/xchacha20/.deps/libsodium_la-stream_xchacha20.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_stream/xchacha20/stream_xchacha20.c -fPIE -o crypto_stream/xchacha20/libsodium_la-stream_xchacha20.o
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT randombytes/sysrandom/libsodium_la-randombytes_sysrandom.lo -MD -MP -MF randombytes/sysrandom/.deps/libsodium_la-randombytes_sysrandom.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/randombytes/sysrandom/randombytes_sysrandom.c -fPIE -o randombytes/sysrandom/libsodium_la-randombytes_sysrandom.o
mv -f crypto_stream/salsa2012/ref/.deps/libsodium_la-stream_salsa2012_ref.Tpo crypto_stream/salsa2012/ref/.deps/libsodium_la-stream_salsa2012_ref.Plo
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT randombytes/internal/librdrand_la-randombytes_internal_random.lo -MD -MP -MF randombytes/internal/.deps/librdrand_la-randombytes_internal_random.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/randombytes/internal/randombytes_internal_random.c -fPIE -o randombytes/internal/librdrand_la-randombytes_internal_random.o
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT crypto_stream/salsa20/xmm6int/libsse2_la-salsa20_xmm6int-sse2.lo -MD -MP -MF crypto_stream/salsa20/xmm6int/.deps/libsse2_la-salsa20_xmm6int-sse2.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_stream/salsa20/xmm6int/salsa20_xmm6int-sse2.c -fPIE -o crypto_stream/salsa20/xmm6int/libsse2_la-salsa20_xmm6int-sse2.o
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT crypto_onetimeauth/poly1305/sse2/libsse2_la-poly1305_sse2.lo -MD -MP -MF crypto_onetimeauth/poly1305/sse2/.deps/libsse2_la-poly1305_sse2.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_onetimeauth/poly1305/sse2/poly1305_sse2.c -fPIE -o crypto_onetimeauth/poly1305/sse2/libsse2_la-poly1305_sse2.o
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT crypto_aead/aes256gcm/aesni/libaesni_la-aead_aes256gcm_aesni.lo -MD -MP -MF crypto_aead/aes256gcm/aesni/.deps/libaesni_la-aead_aes256gcm_aesni.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_aead/aes256gcm/aesni/aead_aes256gcm_aesni.c -fPIE -o crypto_aead/aes256gcm/aesni/libaesni_la-aead_aes256gcm_aesni.o
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT crypto_pwhash/scryptsalsa208sha256/sse/libsse2_la-pwhash_scryptsalsa208sha256_sse.lo -MD -MP -MF crypto_pwhash/scryptsalsa208sha256/sse/.deps/libsse2_la-pwhash_scryptsalsa208sha256_sse.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_pwhash/scryptsalsa208sha256/sse/pwhash_scryptsalsa208sha256_sse.c -fPIE -o crypto_pwhash/scryptsalsa208sha256/sse/libsse2_la-pwhash_scryptsalsa208sha256_sse.o
libtool: compile:  C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/clang.exe -DPACKAGE_NAME=\"libsodium\" -DPACKAGE_TARNAME=\"libsodium\" -DPACKAGE_VERSION=\"1.0.19\" "-DPACKAGE_STRING=\"libsodium 1.0.19\"" -DPACKAGE_BUGREPORT=\"https://github.com/jedisct1/libsodium/issues\" -DPACKAGE_URL=\"https://libsodium.org\" -DPACKAGE=\"libsodium\" -DVERSION=\"1.0.19\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DHAVE_WCHAR_H=1 -DSTDC_HEADERS=1 -D_ALL_SOURCE=1 -D_DARWIN_C_SOURCE=1 -D_GNU_SOURCE=1 -D_HPUX_ALT_XOPEN_SOCKET_API=1 -D_NETBSD_SOURCE=1 -D_OPENBSD_SOURCE=1 -D_POSIX_PTHREAD_SEMANTICS=1 -D__STDC_WANT_IEC_60559_ATTRIBS_EXT__=1 -D__STDC_WANT_IEC_60559_BFP_EXT__=1 -D__STDC_WANT_IEC_60559_DFP_EXT__=1 -D__STDC_WANT_IEC_60559_FUNCS_EXT__=1 -D__STDC_WANT_IEC_60559_TYPES_EXT__=1 -D__STDC_WANT_LIB_EXT2__=1 -D__STDC_WANT_MATH_SPEC_FUNCS__=1 -D_TANDEM_SOURCE=1 -D__EXTENSIONS__=1 -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DHAVE_C_VARARRAYS=1 -DTLS=_Thread_local -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ARMCRYPTO=1 -DHAVE_SYS_MMAN_H=1 -DHAVE_SYS_PARAM_H=1 -DHAVE_SYS_RANDOM_H=1 -DHAVE_SYS_AUXV_H=1 -DHAVE_CET_H=1 -DNATIVE_LITTLE_ENDIAN=1 -DHAVE_INLINE_ASM=1 -DHAVE_TI_MODE=1 -DASM_HIDE_SYMBOL=.hidden -DHAVE_WEAK_SYMBOLS=1 -DHAVE_ATOMIC_OPS=1 -DHAVE_C11_MEMORY_FENCES=1 -DHAVE_GCC_MEMORY_FENCES=1 -DHAVE_ALLOCA_H=1 -DHAVE_ALLOCA=1 -DHAVE_ARC4RANDOM=1 -DHAVE_ARC4RANDOM_BUF=1 -DHAVE_MMAP=1 -DHAVE_MLOCK=1 -DHAVE_MADVISE=1 -DHAVE_MPROTECT=1 -DHAVE_RAISE=1 -DHAVE_SYSCONF=1 -DHAVE_GETPID=1 -DHAVE_GETAUXVAL=1 -DHAVE_POSIX_MEMALIGN=1 -DHAVE_NANOSLEEP=1 -DHAVE_CLOCK_GETTIME=1 -DCONFIGURED=1 -I. -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium -I../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/include/sodium -I./include/sodium --target=aarch64-none-linux-android21 -DANDROID -D_FORTIFY_SOURCE=2 --sysroot=C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/sysroot -g -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -Wformat -Werror=format-security -fPIC -fno-limit-debug-info -pthread -fvisibility=hidden -fPIC -fno-strict-aliasing -fno-strict-overflow -fstack-protector -Wno-deprecated-declarations -Wno-unknown-pragmas -ftls-model=local-dynamic -MT crypto_stream/chacha20/dolbeau/libssse3_la-chacha20_dolbeau-ssse3.lo -MD -MP -MF crypto_stream/chacha20/dolbeau/.deps/libssse3_la-chacha20_dolbeau-ssse3.Tpo -c ../.././../src/1.0.19-fdaa0be457.clean/src/libsodium/crypto_stream/chacha20/dolbeau/chacha20_dolbeau-ssse3.c -fPIE -o crypto_stream/chacha20/dolbeau/libssse3_la-chacha20_dolbeau-ssse3.o
mv -f crypto_secretbox/xchacha20poly1305/.deps/libsodium_la-secretbox_xchacha20poly1305.Tpo crypto_secretbox/xchacha20poly1305/.deps/libsodium_la-secretbox_xchacha20poly1305.Plo
mv -f crypto_stream/salsa208/ref/.deps/libsodium_la-stream_salsa208_ref.Tpo crypto_stream/salsa208/ref/.deps/libsodium_la-stream_salsa208_ref.Plo
mv -f crypto_scalarmult/ristretto255/ref10/.deps/libsodium_la-scalarmult_ristretto255_ref10.Tpo crypto_scalarmult/ristretto255/ref10/.deps/libsodium_la-scalarmult_ristretto255_ref10.Plo
mv -f crypto_stream/xchacha20/.deps/libsodium_la-stream_xchacha20.Tpo crypto_stream/xchacha20/.deps/libsodium_la-stream_xchacha20.Plo
mv -f crypto_stream/salsa20/xmm6int/.deps/libsse2_la-salsa20_xmm6int-sse2.Tpo crypto_stream/salsa20/xmm6int/.deps/libsse2_la-salsa20_xmm6int-sse2.Plo
mv -f randombytes/sysrandom/.deps/libsodium_la-randombytes_sysrandom.Tpo randombytes/sysrandom/.deps/libsodium_la-randombytes_sysrandom.Plo
mv -f crypto_onetimeauth/poly1305/sse2/.deps/libsse2_la-poly1305_sse2.Tpo crypto_onetimeauth/poly1305/sse2/.deps/libsse2_la-poly1305_sse2.Plo
mv -f randombytes/internal/.deps/librdrand_la-randombytes_internal_random.Tpo randombytes/internal/.deps/librdrand_la-randombytes_internal_random.Plo
mv -f crypto_stream/chacha20/dolbeau/.deps/libssse3_la-chacha20_dolbeau-ssse3.Tpo crypto_stream/chacha20/dolbeau/.deps/libssse3_la-chacha20_dolbeau-ssse3.Plo
mv -f crypto_pwhash/scryptsalsa208sha256/sse/.deps/libsse2_la-pwhash_scryptsalsa208sha256_sse.Tpo crypto_pwhash/scryptsalsa208sha256/sse/.deps/libsse2_la-pwhash_scryptsalsa208sha256_sse.Plo
mv -f crypto_aead/aes256gcm/aesni/.deps/libaesni_la-aead_aes256gcm_aesni.Tpo crypto_aead/aes256gcm/aesni/.deps/libaesni_la-aead_aes256gcm_aesni.Plo
libtool: link: C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/llvm-ar.exe cr .libs/libavx512f.a  crypto_pwhash/argon2/libavx512f_la-argon2-fill-block-avx512f.o
libtool: link: C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/llvm-ar.exe cr .libs/libsse41.a  crypto_generichash/blake2b/ref/libsse41_la-blake2b-compress-sse41.o
libtool: link: C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/llvm-ar.exe cr .libs/libavx2.a  crypto_generichash/blake2b/ref/libavx2_la-blake2b-compress-avx2.o crypto_pwhash/argon2/libavx2_la-argon2-fill-block-avx2.o crypto_stream/chacha20/dolbeau/libavx2_la-chacha20_dolbeau-avx2.o crypto_stream/salsa20/xmm6int/libavx2_la-salsa20_xmm6int-avx2.o
libtool: link: C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/llvm-ranlib.exe .libs/libavx512f.a
libtool: link: C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/llvm-ranlib.exe .libs/libsse41.a
libtool: link: C:/Users/neuro/AppData/Local/Android/Sdk/ndk/26.2.11394342/toolchains/llvm/prebuilt/windows-x86_64/bin/llvm-ranlib.exe .libs/libavx2.a
libtool: link: ( cd ".libs" && rm -f "libavx512f.la" && cp -pR "../libavx512f.la" "libavx512f.la" )
libtool: link: ( cd ".libs" && rm -f "libsse41.la" && cp -pR "../libsse41.la" "libsse41.la" )
libtool: link: ( cd ".libs" && rm -f "libavx2.la" && cp -pR "../libavx2.la" "libavx2.la" )
make[3]: Leaving directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src/libsodium'
make[2]: Leaving directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src/libsodium'
make[1]: Leaving directory '/d/sandbox/sandbox/toolchain/vcpkg/buildtrees/libsodium/arm64-android-dbg/src'
```
</details>

--- a/ports/libsodium/arm-neon.diff
+++ b/ports/libsodium/arm-neon.diff
@@ -1,0 +1,119 @@
+diff --git a/configure.ac b/configure.ac
+index df83ef5..be67d34 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -398,11 +398,6 @@ AS_IF([test "x$EMSCRIPTEN" = "x"], [
+     have_armcrypto=no
+     AC_MSG_CHECKING(for ARM crypto instructions set)
+     AC_LINK_IFELSE([AC_LANG_PROGRAM([[
+-      #ifdef __clang__
+-      #  pragma clang attribute push(__attribute__((target("neon,crypto,aes"))), apply_to = function)
+-      #elif defined(__GNUC__)
+-      #  pragma GCC target("+simd+crypto")
+-      #endif
+       #ifndef __ARM_FEATURE_CRYPTO
+       #  define __ARM_FEATURE_CRYPTO 1
+       #endif
+@@ -411,6 +406,12 @@ AS_IF([test "x$EMSCRIPTEN" = "x"], [
+       #endif
+ 
+       #include <arm_neon.h>
++
++      #ifdef __clang__
++      #  pragma clang attribute push(__attribute__((target("neon,crypto,aes"))), apply_to = function)
++      #elif defined(__GNUC__)
++      #  pragma GCC target("+simd+crypto")
++      #endif
+     ]], [[
+       vaeseq_u8(vmovq_n_u8(0), vmovq_n_u8(0));
+ 
+diff --git a/src/libsodium/crypto_aead/aegis128l/aegis128l_armcrypto.c b/src/libsodium/crypto_aead/aegis128l/aegis128l_armcrypto.c
+index 825de8a..a01f60c 100644
+--- a/src/libsodium/crypto_aead/aegis128l/aegis128l_armcrypto.c
++++ b/src/libsodium/crypto_aead/aegis128l/aegis128l_armcrypto.c
+@@ -17,12 +17,6 @@
+ 
+ #include "aegis128l_armcrypto.h"
+ 
+-#ifdef __clang__
+-#pragma clang attribute push(__attribute__((target("neon,crypto,aes"))), apply_to = function)
+-#elif defined(__GNUC__)
+-#pragma GCC target("+simd+crypto")
+-#endif
+-
+ #ifndef __ARM_FEATURE_CRYPTO
+ #define __ARM_FEATURE_CRYPTO 1
+ #endif
+@@ -32,6 +26,12 @@
+ 
+ #include <arm_neon.h>
+ 
++#ifdef __clang__
++#pragma clang attribute push(__attribute__((target("neon,crypto,aes"))), apply_to = function)
++#elif defined(__GNUC__)
++#pragma GCC target("+simd+crypto")
++#endif
++
+ #define AES_BLOCK_LENGTH 16
+ 
+ typedef uint8x16_t aes_block_t;
+diff --git a/src/libsodium/crypto_aead/aegis256/aegis256_armcrypto.c b/src/libsodium/crypto_aead/aegis256/aegis256_armcrypto.c
+index e1ebd57..058e207 100644
+--- a/src/libsodium/crypto_aead/aegis256/aegis256_armcrypto.c
++++ b/src/libsodium/crypto_aead/aegis256/aegis256_armcrypto.c
+@@ -17,12 +17,6 @@
+ 
+ #include "aegis256_armcrypto.h"
+ 
+-#ifdef __clang__
+-#pragma clang attribute push(__attribute__((target("neon,crypto,aes"))), apply_to = function)
+-#elif defined(__GNUC__)
+-#pragma GCC target("+simd+crypto")
+-#endif
+-
+ #ifndef __ARM_FEATURE_CRYPTO
+ #define __ARM_FEATURE_CRYPTO 1
+ #endif
+@@ -32,6 +26,12 @@
+ 
+ #include <arm_neon.h>
+ 
++#ifdef __clang__
++#pragma clang attribute push(__attribute__((target("neon,crypto,aes"))), apply_to = function)
++#elif defined(__GNUC__)
++#pragma GCC target("+simd+crypto")
++#endif
++
+ #define AES_BLOCK_LENGTH 16
+ 
+ typedef uint8x16_t aes_block_t;
+diff --git a/src/libsodium/crypto_aead/aes256gcm/armcrypto/aead_aes256gcm_armcrypto.c b/src/libsodium/crypto_aead/aes256gcm/armcrypto/aead_aes256gcm_armcrypto.c
+index 0a5a128..aa76f5c 100644
+--- a/src/libsodium/crypto_aead/aes256gcm/armcrypto/aead_aes256gcm_armcrypto.c
++++ b/src/libsodium/crypto_aead/aes256gcm/armcrypto/aead_aes256gcm_armcrypto.c
+@@ -19,12 +19,6 @@
+ #define __vectorcall
+ #endif
+ 
+-#ifdef __clang__
+-#pragma clang attribute push(__attribute__((target("neon,crypto,aes"))), apply_to = function)
+-#elif defined(__GNUC__)
+-#pragma GCC target("+simd+crypto")
+-#endif
+-
+ #ifndef __ARM_FEATURE_CRYPTO
+ #define __ARM_FEATURE_CRYPTO 1
+ #endif
+@@ -34,6 +28,12 @@
+ 
+ #include <arm_neon.h>
+ 
++#ifdef __clang__
++#pragma clang attribute push(__attribute__((target("neon,crypto,aes"))), apply_to = function)
++#elif defined(__GNUC__)
++#pragma GCC target("+simd+crypto")
++#endif
++
+ #define ABYTES    crypto_aead_aes256gcm_ABYTES
+ #define NPUBBYTES crypto_aead_aes256gcm_NPUBBYTES
+ #define KEYBYTES  crypto_aead_aes256gcm_KEYBYTES

--- a/ports/libsodium/portfile.cmake
+++ b/ports/libsodium/portfile.cmake
@@ -4,6 +4,8 @@ vcpkg_from_github(
     REF ${VERSION}
     SHA512 6094d7bf191ea3be85f2ddab76b71f1b9c69c786493db5b84d3c5d5a0237003377ddf6a8687a962ea651fe4a9369cf5ee1676ba0bae82690f5f7ef31a698efa9
     HEAD_REF master
+    PATCHES
+        arm-neon.diff # https://github.com/jedisct1/libsodium/commit/8f453f41f8834e0fe47610f2a3e03e696ddb3450 with fuzz 2
 )
 
 if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)

--- a/ports/libsodium/vcpkg.json
+++ b/ports/libsodium/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libsodium",
   "version": "1.0.19",
-  "port-version": 1,
+  "port-version": 2,
   "description": "A modern and easy-to-use crypto library",
   "homepage": "https://libsodium.org/",
   "license": "ISC",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4938,7 +4938,7 @@
     },
     "libsodium": {
       "baseline": "1.0.19",
-      "port-version": 1
+      "port-version": 2
     },
     "libsonic": {
       "baseline": "0.2.0",

--- a/versions/l-/libsodium.json
+++ b/versions/l-/libsodium.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "24563a7817fe403090f82593e4be4c3bc8219510",
+      "version": "1.0.19",
+      "port-version": 2
+    },
+    {
       "git-tree": "38de4073e90ae48d9d7bc68db71b850b5645f5ab",
       "version": "1.0.19",
       "port-version": 1

